cmake/branding/logo-small.svg

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
I don't know the history behind the various SVG files, but I recommend the following:
1. Export all SVGs to be optimized for web consumption.
2. If an SVG is too small, it should be just a few lines in the SVG file to change the natural size of it, which may alleviate having multiple copies of the same scalable file (assuming that's what these files are, they don't load in GitHub when I try to preview them).

Reply:
They aren't the same file, one has a thicker border. They render to the images found in [`/cmake/nsis/assets/`](https://github.com/LMMS/lmms/tree/master/cmake/nsis/assets).

As far as optimization goes, I don't know a whole lot about it right now, but I did make sure to remove anything and everything from the files that wasn't essential for it to render. All of them show up in browsers correctly with the exception of the triangle pattern on the splash screen.

Reply:
Ok, I'll need @Umcaruje / @RebeccaDeField's input on that then.  I knew they were working on pixel-perfect icons, but when we're batching the `.icns` at multiple resolutions using an SVG, it's only appropriate to ask why that same technique isn't identical across platforms.

Reply:
Those two files, `logo.svg` and `logo-small.svg` are *only* for NSIS and aren't used anywhere else. They are also completely flat without a gradient and don't have rounded corners. These files do not end up in any icon, they are images for the NSIS installer.

Reply:
> they exist solely for the Windows NSIS installer.

This statement is not entirely true.  They exist as the desktop icon for LMMS on Windows, which -- as questioned, I'd assume can use the same SVG as the others, no?

The NSIS installer and the `lmms.exe` share this `ico` file.  Both it and the project icon are embedded using the RC resource file here: 
https://github.com/LMMS/lmms/blob/ce5c74432e83f0eaa617cc26332f4209bc4efc7f/cmake/nsis/lmms.rc.in#L1

It's supplied to CMake here: https://github.com/LMMS/lmms/blob/ce5c74432e83f0eaa617cc26332f4209bc4efc7f/src/CMakeLists.txt#L40

Reply:
The suspect files are not icons and **never become icons** of any sort. They are used for the images within the NSIS window. The windows `.ico` is handled cleanly just like the rest of the platforms.

Reply:
... so if there are visual differences between the Windows file and the file used for other platforms, I assume @Umcaruje or @RebeccaDeField know as they recently added these files as part of the 1.2.0 rebranding (unless you know?)

Reply:
I just ran the NSIS installer to double-check, and I actually have no idea what those files are for. They don't appear in the installer window at all. In fact, it isn't referenced at all by CMakeLists.txt! I think we definitely need the inputs of the artists to understand the actual purpose of those images.

I still must stress, however, **those two files are not icons and never become icons.** They are part of the recoloring script because I was under the impression that they were part of the installer window branding. If the files aren't used, they will be removed from the script completely.

Reply:
@iansannar sorry, I'm terribly sorry for any confusion, I see them now.  They've something to do with Windows 8/10 modern start menu.  @Umcaruje added/explains them a bit here: https://github.com/LMMS/lmms/pull/4519#issuecomment-423307928, used by NSIS here: https://github.com/LMMS/lmms/blob/ce5c74432e83f0eaa617cc26332f4209bc4efc7f/cmake/nsis/lmms.VisualElementsManifest.xml#L4-L6.

What you're doing now to recolor them is perfect.  Sorry to have wasted your time. Those file names threw me off.

Reply:
It's all good! It's a relief to know what they are for. The mystery has been solved! 😀

Reply:
Perhaps we should give these Windows 10 SVGs names that add context to their purpose, e.g. `logo-uwp-small.svg`, where UWP = Universal Windows Platform, the home for [VisualElements](https://docs.microsoft.com/en-us/uwp/schemas/appxpackage/appxmanifestschema/element-visualelements) standard for these weird tiles.